docs/pages/server-access/guides/openssh.mdx

```diff
@@ -0,0 +1,348 @@
+---
+title: Using Teleport with OpenSSH
+description: How to use Teleport on legacy systems with OpenSSH and sshd.
+---
+
+Teleport is fully compatible with OpenSSH and can be quickly set up to record and
+audit all SSH activity. Using Teleport and OpenSSH has the advantage of getting you up
+and running, but in the long run, we would recommend replacing `sshd` with `teleport`.
+We've outlined these reasons in [OpenSSH vs Teleport SSH for Servers?](https://gravitational.com/blog/openssh-vs-teleport/)
+
+Teleport is a standards-compliant SSH proxy and it can work in environments with
+existing SSH implementations, such as OpenSSH. This section will cover:
+
+- Configuring OpenSSH server `sshd` to join a Teleport cluster. Existing fleets of
+  OpenSSH servers can be configured to accept SSH certificates dynamically issued by a Teleport CA.
+- Configuring OpenSSH client `ssh` to login into nodes inside a Teleport
+  cluster.
+
+<Admonition
+  type="warning"
+  title="Warning"
+>
+  The minimum OpenSSH version which will work with Teleport is version 6.9.
+  You can view your OpenSSH version with `ssh -V`.
+</Admonition>
+
+<Admonition type="warning" title="Warning">
+  The examples below may include the use of the `sudo` keyword to make following each step easier when creating resources from scratch. However, we generally discourage the use of `sudo` in **real-world production environments** per the *Principle of Least Privilege* (POLP). 
+</Admonition>
+
+### Follow along with our video guide
+
+<iframe
+  width="712"
+  height="400"
+  src="https://www.youtube-nocookie.com/embed/x0eYFUEIOrM?rel=0&modestbranding=1"
+  frameBorder="0"
+  allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
+  allowFullScreen
+/>
+
+## Architecture
+
+<Figure
+  align="center"
+  bordered
+  caption="Teleport OpenSSH Recording Proxy"
+>
+  ![Teleport OpenSSH Recording Proxy](../../../img/openssh-proxy.svg)
```

Comment:
In this diagram, it's not clear what's encrypted and what's decrypted for me. You can replace with `Terminate SSH` (record) -> Reestablish SSH

and `Forward agent` and it will help readers familiar with open ssh to understand how it works.

Reply:
This is being handled here: https://github.com/gravitational/web/issues/1774 but I'll include an updated placeholder.